README.md

```diff
@@ -1,7 +1,117 @@
 python-gp
 ===========
+[![Python3.6](https://img.shields.io/badge/python-3.6-red.svg)](https://www.python.org/downloads/release/python-369/)
+[![Python3.7](https://img.shields.io/badge/python-3.7-red.svg)](https://www.python.org/)
+[![Python3.8](https://img.shields.io/badge/python-3.8-red.svg)](https://www.python.org/)
+[![GPL license](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-3.0.html)
+
+Cartesian genetic programming (CGP) in Python.
+
+This library implements Cartesian genetic programming in pure Python. It implements Python data structures to represent computational graphs in the Cartesian Genetic programming and provides one evolutionary algorithm,  (mu + lambda) evolution strategies, to evolve a population of computational graphs given an objective function.
+
+![CGP Sketch](cgp-sketch.png)
+
+Basic usage
+===========
+
+A complete example of CGP applied to a regression problem can be found in `examples/example_evo_regression.py`.
+
+The basic steps to be taken are:
+
+1. **Define an objective function**. 
+
+   The objective needs to take an individual as input variable and update the `fitness` member of the individual.
+```
+def objective(individual):
+    """Objective function of the regression task.
+
+    Parameters
+    ----------
+    individual : gp.CGPIndividual
+        Individual of the Cartesian Genetic Programming Framework.
+
+    Returns
+    -------
+    gp.CGPIndividual
+        Modified individual with updated fitness value.
+    """
+	# Compute the fitness value
+	fitness = ...
+	individual.fitness = fitness
+	return individual
+```
+2. Define parameters for the genome, the population and the evolutionary algorithm
```

Comment:
either all headings should be bold or none

Reply:
I unbolded the first heading.